src/test/mir-opt/const_allocation3/rustc.main.ConstProp.after.mir

```diff
@@ -0,0 +1,43 @@
+// MIR for `main`
+// source = MirSource { instance: Item(DefId(0:3 ~ const_allocation3[317d]::main[0])), promoted: None }
+// pass_name = ConstProp
+// disambiguator = after
+
+fn  main() -> () {
+    let mut _0: ();                      // return place in scope 0 at $DIR/const_allocation3.rs:4:11: 4:11
+    let _1: &Packed;                     // in scope 0 at $DIR/const_allocation3.rs:5:5: 5:8
+    let mut _2: &&Packed;                // in scope 0 at $DIR/const_allocation3.rs:5:5: 5:8
+
+    bb0: {
+        StorageLive(_1);                 // bb0[0]: scope 0 at $DIR/const_allocation3.rs:5:5: 5:8
+        StorageLive(_2);                 // bb0[1]: scope 0 at $DIR/const_allocation3.rs:5:5: 5:8
+        _2 = const Scalar(alloc2+0): &&Packed; // bb0[2]: scope 0 at $DIR/const_allocation3.rs:5:5: 5:8
+                                         // ty::Const
+                                         // + ty: &&Packed
+                                         // + val: Value(Scalar(alloc2+0))
+                                         // mir::Constant
+                                         // + span: $DIR/const_allocation3.rs:5:5: 5:8
+                                         // + literal: Const { ty: &&Packed, val: Value(Scalar(alloc2+0)) }
+        _1 = (*_2);                      // bb0[3]: scope 0 at $DIR/const_allocation3.rs:5:5: 5:8
+        StorageDead(_2);                 // bb0[4]: scope 0 at $DIR/const_allocation3.rs:5:8: 5:9
+        StorageDead(_1);                 // bb0[5]: scope 0 at $DIR/const_allocation3.rs:5:8: 5:9
+        _0 = ();                         // bb0[6]: scope 0 at $DIR/const_allocation3.rs:4:11: 6:2
+        return;                          // bb0[7]: scope 0 at $DIR/const_allocation3.rs:6:2: 6:2
+    }
+}
+
+alloc2 (const_allocation3[317d]::FOO[0]) size: 8, align: 8 {
+    ├──────alloc8+0───────┤ 
+}
+
+alloc8 size: 140, align: 1 {
+    0x00 │ ab ab ab ab ab ab ab ab ab ab ab ab ab ab ab ab ab ab ab ab ab ab ab ab ab ab ab ab ├──alloc6+0──
+    0x20 │ ──────────┤ 01 ef cd ab 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 
```

Comment:
Ah so it prints text on the first line, centered, and then nothing on the second line?

What if there's no enough space  (like only having 1 byte on the first line)?

(FWIW, I'd be fine leaving line-wrapping to a future PR. With relocations being inline now, long lines that are line-wrapped by the terminal / GH don't look quite so bad any more.)

Reply:
> Ah so it prints text on the first line, centered, and then nothing on the second line?

yes

> What if there's no enough space (like only having 1 byte on the first line)?

The text just goes outside the right end

Reply:
> (FWIW, I'd be fine leaving line-wrapping to a future PR. With relocations being inline now, long lines that are line-wrapped by the terminal / GH don't look quite so bad any more.)

Now it's implemented ;)

Reply:
> The text just goes outside the right end

Hm I see. When the second line isn't full, that lacks a clear indication that the width does not match the number of bytes it occupies.

> Now it's implemented ;)

Yeah but there are some outstanding problems. ;)